internal/core/constants.go

```diff
@@ -133,6 +133,22 @@ func StringToHeuristicType(stringType string) HeuristicType {
 	}
 }
 
+// AlertPolicy ... The alerting policy for a heuristic session
+type AlertPolicy struct {
+	Dest string `json:"destination"`
```

Comment:
Is an alert policy string the behavior we want here? What if someone wants to alert multiple sources with one heuristic? This seems limiting

Reply:
https://github.com/base-org/pessimism/issues/81